### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by ryanolsonx** [nkt/atom-autocomplete-modules#109]
Argument to `path.dirname` must be a string
```
Object.dirname (/Applications/Atom.app/Contents/Resources/app.asar/src/electron-shims.js:9:10)
LocalLookup.getModuleDir (/Users/ryan/.atom/packages/autocomplete-modules/lib/utils/path-helpers.js:19:26)
LocalLookup.lookup (/Users/ryan/.atom/packages/autocomplete-modules/lib/lookups/module/local.js:15:32)
LocalLookup.getList (/Users/ryan/.atom/packages/autocomplete-modules/lib/lookups/module/local.js:11:17)
relevantLookups.reduce (/Users/ryan/.atom/packages/autocomplete-modules/lib/lookups/index.js:25:16)
Array.reduce (null:null:null)
```

**Comment by jonyeezs** [comment]
Hi @ryanolsonx could you help give bit more context on how you got the error?

Was this on an existing file or a newly created file?

**Comment by ryanolsonx** [comment]
This message "Object.dirname is deprecated." showed up in my deprecation cop list in Atom. It appears that this method is deprecated. I didn't have to follow any steps to get this to show up. It had a quick link to create this issue.

**Comment by mehdihz** [comment]
I have this deprecated warning. How to fix that warning?

![screen shot 2018-11-03 at 11 56 13 am](https://user-images.githubusercontent.com/1472329/47949899-a9e45200-df5f-11e8-8869-60d34b97ca11.png)

**Comment by jonyeezs** [comment]
Resolved by commit dff6af8a43805e980b7c4a145375b10de3f2d308 in v2.2.6.

**Comment by maretana** [comment]
> Resolved by commit [dff6af8](https://github.com/nkt/atom-autocomplete-modules/commit/dff6af8a43805e980b7c4a145375b10de3f2d308) in v2.2.6.

I can confirm problem is solved. Awesome, thanks!
